back/config/environments/development.rb

```diff
@@ -96,4 +107,15 @@
     # Bullet.stacktrace_includes = [ 'your_gem', 'your_middleware' ]
     # Bullet.stacktrace_excludes = [ 'their_gem', 'their_middleware', ['my_file.rb', 'my_method'], ['my_file.rb', 16..20] ]
   end
+
+  # rubocop:disable Rails/Output
+  console do
+    Object.include ConsoleMethods
```

Comment:
Smart 💡 as an alternative, would it make sense to use:
```suggestion
    extend ConsoleMethods
```
to avoid polluting `Object`?

Reply:
`extend ConsoleMethods` inside `console do` does make the methods available in the console. I tried it. However, doing `extend ConsoleMethods` by hand in the console makes the methods available. 🤔 More research revealed that there is another way, which I applied.

Reply:
Isn't a *not* missing somewhere in your last comment?

Reply:
Yes. I fixed it.